docs/index.rst

```diff
@@ -18,8 +15,12 @@ Features
 * :ref:`Create Versioned Repositories <versioned-repo-created>` so every operation is a restorable snapshot
 * :ref:`Download content on-demand <create-remote>` when requested by clients to reduce disk space
 * :ref:`Perform docker/podman pull <host>` from a container distribution served by Pulp
+* :ref:`Perform docker/podman push <push-workflow>` to the Pulp Registry
 * Curate container images by :ref:`filtering <create-remote>` what is mirrored from an external repository
 * Curate container images by creating repository versions with :ref:`a specific set <recursive-add>` of images
+* :ref:`Build an OCI format image from a Containerfile <build-Containerfile-workflow>` and make it available from Pulp Registry
```

Comment:
Since we do have just one registry and you have already wrote above `Perform docker/podman push <push-workflow> to the Pulp Registry`, I think we should add the definite article here as well.
```suggestion
* :ref:`Build an OCI format image from a Containerfile <build-Containerfile-workflow>` and make it available from the Pulp Registry
```